### PR TITLE
#141 singlelineinput margin

### LINKED ISSUE #141: Fix margin error on SingleLineInput
SingleLineInput component has a minor bottom margin error

### PR DESCRIPTION
**Pull request by joshxg**
Fixes #141 and #142 

Minor fix to margins and padding

### CODE REVIEW

File: src/ui/components/widgets/SingleLineInput.js [hunk 1]
```diff
@@ -9,14 +9,14 @@ const SingleLineInput = props => {
 
   return (
     <div className={classes.container}>
-      <Typography color="textSecondary" variant="button">
+      <Typography color="textSecondary" variant="button" style={{ margin: 'auto' }}>
```
Comment (wlthomson) [inline src/ui/components/widgets/SingleLineInput.js]: Could we add a `typography` class to localStyles? It's only a one-liner, but I think it's good to keep styles as centralised as possible.

**Comment by wlthomson** [comment]
Looks good. See style comment above.

File: src/ui/components/widgets/SingleLineInput.js (hunk 1 above)
Comment (joshxg) [reply src/ui/components/widgets/SingleLineInput.js]: cool done

**Comment by wlthomson** [comment]
Looks good. Merging and closing.

**Pull request merged**